ydb/core/persqueue/partition_write.cpp

```diff
@@ -939,7 +939,9 @@ TPartition::ProcessResult TPartition::ProcessRequest(TWriteMsg& p, ProcessParame
                 );
 
                 TabletCounters.Cumulative()[COUNTER_PQ_WRITE_ALREADY].Increment(1);
+                MsgsDiscarded.Inc();
```

Comment:
Is it correct to Inc MsgsDiscarded when this is mirrored partition? In this case MsgsDiscarded will be inited with default constructor.

Reply:
Yes. Inc() of default-constructed multi-counter makes no effect.